### PR TITLE
Split IndentWithTab and OutdentWhenShiftTab to be able to disable the Alt+Shift+Arrow hotkey 

### PR DESCRIPTION
**Pull request by BryanValverdeU**
This PR add the following functionality: 
- Remove the functionality of `Alt+Shift+Right` from `IndentWithTab` Content edit feature and move it to a new ContentEditFeature called IndentWhenAltShiftRight, so consumers can disable the hotkeys separately.
- Same change for `OutdentWitShiftTab` and `Alt+Shift+Left`. So users can disable the Alt+Shift+Left hotkey

### CODE REVIEW

File: demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts [hunk 1]
```diff
@@ -2,10 +2,14 @@ import { ContentEditFeatureSettings } from 'roosterjs-editor-types';
...
-    return getObjectKeys(allFeatures).reduce((settings, key) => {
-        settings[key] = !allFeatures[key].defaultDisabled;
-        return settings;
-    }, <ContentEditFeatureSettings>{});
+    return {
+        ...getObjectKeys(allFeatures).reduce((settings, key) => {
+            settings[key] = !allFeatures[key].defaultDisabled;
+            return settings;
+        }, <ContentEditFeatureSettings>{}),
+        indentWhenAltShiftRight: true,
```
Comment (juliaroldi) [inline demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts:12]: Why change this?

Comment (BryanValverdeU) [reply demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts:12]: To always have these features enabled in the demo page. 
We added defaultDisabled on the Alt+Shift+Right/Left ContentEdit Features when Browser is in Mac.

Comment (BryanValverdeU) [reply demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts:12]: We can remove it so we have these features disabled in Mac if needed.

Comment (juliaroldi) [reply demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts:12]: I think it is ok, since it only in demo site.

**Review by juliaroldi — APPROVED** [review]

File: demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts (hunk 1 above)
Comment (BryanValverdeU) [reply demo/scripts/controls/sidePane/editorOptions/getDefaultContentEditFeatureSettings.ts:12]: Thanks

**Pull request merged**